jenkins/scripts/files/run_integration_tests.sh

```diff
@@ -79,7 +79,7 @@ if [[ ("${TESTS_FOR}" != "e2e_tests" && "${REPO_NAME}" == "metal3-dev-env") ||
   # If we are testing ansible test from metal3-dev-env or e2e from capm3,
   # it will already be cloned to tested_repo
   pushd tested_repo
-elif [[ "${REPO_NAME}" == "cluster-api-provider-metal3" ]]; then
+elif [[ "${TESTS_FOR}" == "e2e_tests" ]]; then
```

Comment:
Could you add a comment explaining this case? I guess it is when we are running e2e but not from CAPM3, right?

Reply:
Currently when we run integration tests from capm3 it clones capm3 repo instead of dev-env check https://jenkins.nordix.org/job/metal3_capm3_v1b1_integration_test_ubuntu/275/

Reply:
done